core/controllers/contributor_dashboard_test.py

```diff
@@ -980,6 +980,9 @@ def setUp(self):
                 })], 'Changes.')
 
     def test_handler_with_invalid_language_code_raise_exception(self):
+        user_services.allow_user_to_review_translation_in_language(
```

Comment:
Why add this here -- how does it change the test?

Reply:
Removed this code now